app/controllers/concerns/api_error_responses.rb

```diff
@@ -4,6 +4,9 @@ module ApiErrorResponses
   included do
     include ActionController::MimeResponds
     rescue_from ActiveRecord::RecordNotFound, with: :record_not_found
+    rescue_from ActionController::RoutingError, with: :record_not_found
+    #rescue_from StandardError, with: :error
```

Comment:
Don't commit comment

Reply:
I wanted to remember that "solution" if I might need it later... :-)